src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -0,0 +1,481 @@
+        },
+      },
+      response: {
+        invocations: {},
+        transition: '_error',
+      },
+    },
+  ],
+  on: {
+    _success: {
+      id: 'B',
+      label: 'B label',
+      status: 'error',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+  },
+  response: {
+    invocations: {},
+    transition: '_success',
+  },
+};
+
+export const singleNodeWithTransitions = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'single',
+  on: {
+    _success: {
+      id: 'A A',
+      label: 'A A label',
+      status: 'unprocessed',
+      type: 'single',
+    },
+    _error: {
+      id: 'A B',
+      label: 'A B label',
+      status: 'unprocessed',
+      type: 'single',
+    },
+    custom1: {
+      id: 'A C',
+      label: 'A C label',
+      status: 'success',
+      type: 'single',
+      response: {
+        invocations: {},
+      },
+    },
+    custom2: {
+      id: 'A D',
+      label: 'A D label',
+      status: 'unprocessed',
+      type: 'single',
+    },
+  },
+  response: {
+    invocations: {},
+    transition: 'custom1',
+  },
+};
+
+export const compositeWithUnprocessedNodeInSubtask = {
+  id: 'A',
+  label: 'A label',
+  status: 'error',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'success',
+      type: 'single',
+      on: {
+        _success: {
+          id: 'A-A A',
+          label: 'A-A A label',
+          status: 'error',
+          type: 'single',
+          response: {
+            invocations: {},
+          },
+        },
+        _error: {
+          id: 'A-A B',
+          label: 'A-A B label',
+          status: 'unprocessed',
+          type: 'single',
+        },
+      },
+      response: {
+        invocations: {},
+        transition: '_success',
+      },
+    },
+  ],
+};
+
+export const compositeWithUnevenSubtasks = {
+  id: 'A',
+  label: 'A label',
+  status: 'success',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'A-A',
+      label: 'A-A label',
+      status: 'success',
+      type: 'single',
+      on: {
+        _success: {
+          id: 'A-A A',
+          label: 'A-A A label',
+          status: 'success',
+          type: 'single',
+          on: {
+            _success: {
+              id: 'A-A A A',
+              label: 'A-A A A label',
+              status: 'success',
+              type: 'single',
+              response: {
+                invocations: {},
+              },
+            },
+          },
+          response: {
+            invocations: {},
+            transition: '_success',
+          },
+        },
+        _fallback: {
+          id: 'A-A B',
+          label: 'A-A B label',
+          status: 'unprocessed',
+          type: 'single',
+        },
+      },
+      response: {
+        invocations: {},
+        transition: '_success',
+      },
+    },
+    {
+      id: 'A-B',
+      label: 'A-B label',
+      status: 'unprocessed',
```

Comment:
Not sure if it can be unprocessed. Is it the case that it is unprocessed here?

Reply:
It's a mistake, it indeed cannot be unprocessed